apps/web/src/trpc/client.tsx

```
@@ -1,56 +1,56 @@
-  if (typeof window === 'undefined') {
-    // Server: always make a new query client
-    return makeQueryClient();
-  }
-  // Browser: use singleton pattern to keep the same query client
-  return (clientQueryClientSingleton ??= makeQueryClient());
+	if (typeof window === "undefined") {
+		// Server: always make a new query client
+		return makeQueryClient();
+	}
+	// Browser: use singleton pattern to keep the same query client
+	return (clientQueryClientSingleton ??= makeQueryClient());
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)